sdk/storage/storage-common/package.json

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
Will this change storage-common to a project and we'll need to release it as a separate package?

Reply:
Yes, that is the intent that it is a separate package and that the internval avro is also a separate package that must be released.